dbcon/joblist/primitivestep.h

```diff
@@ -1366,7 +1366,15 @@ class TupleBPS : public BatchPrimitive, public TupleDeliveryStep
     {
         return true;
     }
-
+    ResourceManager* resourceManager() const
+    {
+      return fRm;
+    }
+    bool runFEonPM()
```

Comment:
Can't we use const specifier for this method?

Reply:
const added